components/ord-service/src/main/java/com/sap/cloud/cmp/ord/service/health/ReadinessHealthIndicator.java

```diff
@@ -3,29 +3,28 @@
 import com.sap.cloud.cmp.ord.service.repository.SchemaRepository;
 import com.sap.cloud.cmp.ord.service.storage.model.SchemaEntity;
 import org.springframework.beans.factory.annotation.Autowired;
+import org.springframework.beans.factory.annotation.Value;
 import org.springframework.boot.actuate.health.Health;
 import org.springframework.boot.actuate.health.HealthIndicator;
 import org.springframework.stereotype.Component;
 
 @Component("schemaCompatibility")
 public class ReadinessHealthIndicator implements HealthIndicator {
     private boolean isHealthy = false;
+
+    @Value("${schema.migration_version}")
```

Comment:
Why is the annotation argument with mixed indentation (`.` and `_`) ? 

It could be a Spring specific but I don't quite remember.

Reply:
It is how you want to store it in the application.yaml

For me it looked good like this:
```
schema:
    migration_version: <value>
```